airtime_mvc/public/js/airtime/preferences/streamsetting.js

```diff
@@ -159,9 +161,12 @@ function setLiveSourceConnectionOverrideListener(){
     });
     
     master_dj_actions.find("#reset").click(function(event){
-        var port = $("#master_harbor_input_port").val();
-        var mount = $("#master_harbor_input_mount_point").val();
-        var url = "http://"+location.hostname+":"+port+"/"+mount;
+        var port = $("#master_source_port").val();
+        var mount = $("#master_source_mount").val();
+        if (mount.charAt(0) != '/') {
```

Comment:
I see what you are saying. I'm not exactly sure what the 'N/A' is supposed to be there for except for if the user deleted both port and mount from the form.